app/assets/javascripts/oxalis/view/settings/user_settings_view.js

```diff
@@ -268,7 +268,7 @@ class UserSettingsView extends PureComponent<UserSettingsViewProps> {
             onChange={this.onChangeUser.overrideNodeRadius}
           />
           <SwitchSetting
-            label="Single-Node-Tree Mode"
+            label={`Allow Single-node-tree mode ("Soma clicking")${""}`}
```

Comment:
The `Allow` doesn't make sense in this case imo. The setting is used to activate the mode (whereas in the other views, the caption is about the allowance of the mode, so there it's fine).